app/generators/questionnaire_generator.rb

```diff
@@ -702,11 +707,10 @@ def generate_unsubscribe_content(question)
     end
 
     def generate_unsubscribe_action(question)
-      response = Response.find_by_id(question[:response_id])
       url_href = '#'
-      url_href = Rails.application.routes.url_helpers.questionnaire_path(uuid: response.uuid) if response
+      url_href = question[:unsubscribe_url] if question[:unsubscribe_url]
       body = content_tag(:a, (question[:button_text] || 'Uitschrijven').html_safe,
-                         'data-method': 'delete',
+                         'data-method': (question[:data_method] || 'delete'),
```

Comment:
Misschien ook even toevoegen aan de readme, dat dit een optie is.

Reply:
Goeie. Fixed.